packages/plugin-corsi-blocks/src/index.ts

```diff
@@ -294,15 +294,14 @@ class CorsiBlocksPlugin implements JsPsychPlugin<Info> {
           display_element
             .querySelector(`.jspsych-corsi-block[data-id="${id}"]`)
             .animate(correct_animation, animation_timing);
-          if (trial_data.response.length == trial.sequence.length) {
-            trial_data.correct = true;
-            setTimeout(end_trial, trial.response_animation_duration); // allows animation to finish
-          }
         } else {
           display_element
             .querySelector(`.jspsych-corsi-block[data-id="${id}"]`)
             .animate(incorrect_animation, animation_timing);
-          trial_data.correct = false;
+        }
```

Comment:
Thank you @Pavel6625 for working on this? I was about to work on this feature as well. This helps our use case as well. I am wondering if this needs to be an optional feature (turned on or off via a feature flag). The old behavior might still be useful under certain circumstances.